README.md

```diff
@@ -6,7 +6,7 @@
 [![FIWARE Core Context Management](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/core.svg)](https://www.fiware.org/developers/catalogue/)
 [![License badge](https://img.shields.io/github/license/telefonicaid/fiware-orion.svg)](https://opensource.org/licenses/AGPL-3.0)
 [![Docker badge](https://img.shields.io/docker/pulls/fiware/orion.svg)](https://hub.docker.com/r/fiware/orion/)
-[![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/fiware-orion)
+[![Support badge](https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow)](http://stackoverflow.com/questions/tagged/fiware-orion)
```

Comment:
I have tried to access directly to the URL https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow and I get:

![imagen](https://user-images.githubusercontent.com/1534240/159237120-1ebc39cd-5105-4c76-9b40-e8915d1f2f94.png)

It seems it is not working...

Reply:
That is weird

```
![](https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow)
```

gives ![](https://img.shields.io/badge/tag-fiware--orion-orange.svg?logo=stackoverflow) for me.

Reply:
Now it's also working for me. Maybe it has been some kind of temporal glitch in img.shields.io

NTC